x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/TextExpansionProcessor.java

```diff
@@ -27,6 +28,8 @@ public class TextExpansionProcessor extends NlpTask.Processor {
     private final NlpTask.RequestBuilder requestBuilder;
     private Map<Integer, String> replacementVocab;
 
+    private boolean chunkResults = true;
```

Comment:
Just checking, is this used? Sorry if I missed it, but it looks like it's shadowed in a couple places below, so I wasn't sure.

Reply:
yup leftover from the first testing I did